### DISCUSSION TITLE
Any plans for making UI/UX better ?

### DISCUSSION THREAD

**Discussion by b00tkitism** [NationalSecurityAgency/ghidra#5837] [Q&A] [answered]

**Comment by astrelsky** [comment, marked as answer]
You're going to have to be more specific.

**Reply by xxr0ss** [reply]
I think what @alom4hdi means is a more modern looking UI/UX for Ghidra

**Comment by graps01** [comment]
I have a suggestion on the User Interface of the main Ghidra disassembly window, or the dragon head selection window: Please continue to use the F1 key for help on general interface navigation. I appreciate the extensive help documentation ! Thanks.

**Reply by Wall-AF** [reply]
Please DON'T suggest the removal of the menu bar. It's very valuable for keyboarders and would be a retrograde step that seems to be proliferating elsewhere!
